tests/backends/data/test_mongo.py

```diff
@@ -852,7 +852,7 @@ def test_backends_data_mongo_data_backend_write_method_with_custom_chunk_size(
     """Test the `MongoDataBackend.write` method, given a custom chunk_size, should
     insert the provided documents to target collection by batches of size `chunk_size`.
     """
-    # pylint: disable=unused-argument
```

Comment:
shouldn't we keep this ?

Reply:
You're right!
Adding `ARG` set of rules (`flake8-unused-arguments`) that should raise warnings for this.

Reply:
After discussion, adding `ARG` set of rules but ignoring the tests files (it is really a pain for fixtures)